docker/cellranger/1.1.0/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Supply a valid download link and md5sum from "https://support.10xgenomics.com/single-cell-gene-expression/software/downloads/latest"
+
+FROM ubuntu:20.04
+
+RUN apt-get update && \
+    apt-get upgrade -y && \
+    apt-get install curl -y && \
+    rm -r /var/lib/apt/lists/*
+
+WORKDIR /opt
+
+ARG cellranger_url
+ARG cellranger_md5
+
+RUN curl -o cellranger.tar.gz \
+    ${cellranger_url} \
+    && echo "${cellranger_md5}  cellranger.tar.gz" > cellranger.tar.gz.md5 \
+    && md5sum -c cellranger.tar.gz.md5 \
+    && tar -xzvf cellranger.tar.gz \
+    && mv cellranger-* cellranger
+
+ENV PATH "/opt/cellranger:$PATH"
```

Comment:
For consistency with our other images, we should add an entrypoint to this image.

Reply:
Done. But I just poked around and not all our images have an ENTRYPOINT. Outside the scope of this PR to address that for other images? I don't like that it's not consistent, but don't think this PR is the place for that fix.

Reply:
Thanks, there may be some images (`util` comes to mind) where an entrypoint does not make sense. But otherwise, I think we should include them. Agree that this should be a separate PR.